docker-compose_mysql.yml

```diff
@@ -0,0 +1,47 @@
+version: "3"
+services:
+  app:
+    env_file:
+      - .env
+    container_name: awesome_nest_boilerplate
+    restart: always
+    build: .
+    ports:
+      - "$PORT:$PORT"
+    links:
+      - mysql
+    environment:
+      DB_HOST: mysql
+  mysql:
+    image: mysql
+    container_name: mysql
+    restart: always
+    environment:
+      ## Note: all of these environment variables can also be loaded from .env
+      
+      MYSQL_DATABASE: 'nest_boilerplate'
+      # So you don't have to use root, but you can if you like
+      MYSQL_USER: 'mysql'
```

Comment:
let's map DB_* variables to MYSQL_*, maybe `MYSQL_USER: $DB_USERNAME` could work, (need some research)